ros_buildfarm/sourcerpm_job.py

```diff
@@ -55,6 +55,22 @@ def build_sourcerpm(
         '--depth', '1', '--no-single-branch',
         release_repository_url, os_pkg_name]
 
+    # HACK FOR RHBZ#1960791
+    # RHEL 8.4 shipped a cmake package which omitted the 'cmake3' virtual
+    # package. This hack replaces dependencies on 'cmake3' with 'cmake>3',
+    # which will work on RHEL 8, but not RHEL 7.
+    if os_name == 'rhel' and os_code_name == '8':
+        fixup_cmd = [
+            'sed', '-i',
```

Comment:
```suggestion
            'sed', '-i',
            '-E',
```

I think you'll need to use extended regular expressions with sed in order to take advantage of capture groups and backreferences. I usually need to.

Reply:
It doesn't seem to be necessary, but there are other advantages to `-E` that I'll comment on in another thread.

Reply:
How utterly peculiar as it is definitely required for backreferences to work on my Arch workstation.